pymc_marketing/mmm/transformers.py

```diff
@@ -908,7 +908,7 @@ def hill_saturation(
     r"""Hill Saturation Function
 
     .. math::
-        f(x) = \frac{\sigma}{1 + e^{-\beta(x - \lambda)}}
+        f(x) = \frac{\sigma}{1 + e^{-\beta(x - \lambda)}} - \frac{\sigma}{1 + e^{\beta\lambda}}
```

Comment:
How does this effect the parameter interpretations that are listed in the docstring below?

Reply:
Ah! I did not look into the docstrings 🙈. I will work on them. In essence we are adding an offset. Should not change that much. 

I do think this has to be corrected as these saturation functions should pass through the origin.

Reply:
Totally!

Reply:
Done!